openshift/kustomize/api/overlays/prod/kustomization.yaml

```diff
@@ -133,7 +133,7 @@ patches:
         value: 1Gi
       - op: replace
         path: /spec/template/spec/containers/0/image
-        value: image-registry.openshift-image-registry.svc:5000/9b301c-tools/api:prod
+        value: image-registry.openshift-image-registry.svc:5000/9b301c-prod/api:prod
```

Comment:
We don't push images to each environment, we keep them all in the tools namespace.

Reply:
Image pull updated back to tools.